iOS/Flows/Home/_New/new_ConversationViewController.swift

```diff
@@ -24,6 +24,8 @@ class new_ConversationViewController: FullScreenViewController,
     // Custom Input Accessory View
     lazy var messageInputAccessoryView = InputAccessoryView(with: self)
 
+    var onSelectedThread: ((ChannelId, MessageId) -> Void)?
```

Comment:
As per our conversation, I'm not using a protocol delegate here, and instead using a closure for delegation.